Build error in public_server tests

Starting with 54c6b089174c704c5d4a09e94d7537ba87ea6cf7 I get this build error:
```
In file included from /home/jdboyd/work/civetweb/test/civetweb_check.h:38:0,
                 from /home/jdboyd/work/civetweb/test/public_server.h:24,
                 from /home/jdboyd/work/civetweb/test/public_server.c:33:
/home/jdboyd/work/civetweb/test/public_server.c: In function ‘test_request_handlers’:
/home/jdboyd/work/cwb/third_party/include/check.h:476:20: error: conversion to ‘intmax_t {aka long int}’ from ‘size_t {aka long unsigned int}’ may change the sign of the result [-Werror=sign-conversion]
   intmax_t _ck_y = (Y); \
                    ^
/home/jdboyd/work/cwb/third_party/include/check.h:557:32: note: in expansion of macro ‘_ck_assert_int’
 #define ck_assert_int_ge(X, Y) _ck_assert_int(X, >=, Y)
                                ^
/home/jdboyd/work/civetweb/test/public_server.c:1082:2: note: in expansion of macro ‘ck_assert_int_ge’
  ck_assert_int_ge(i, strlen(expected_cgi_result));
  ^
/home/jdboyd/work/cwb/third_party/include/check.h:476:20: error: conversion to ‘intmax_t {aka long int}’ from ‘size_t {aka long unsigned int}’ may change the sign of the result [-Werror=sign-conversion]
   intmax_t _ck_y = (Y); \
                    ^
/home/jdboyd/work/cwb/third_party/include/check.h:492:32: note: in expansion of macro ‘_ck_assert_int’
 #define ck_assert_int_eq(X, Y) _ck_assert_int(X, ==, Y)
                                ^
/home/jdboyd/work/civetweb/test/public_server.c:1089:2: note: in expansion of macro ‘ck_assert_int_eq’
  ck_assert_int_eq(i, strlen(expected_cgi_result));
  ^
cc1: all warnings being treated as errors
```

This is on Ubuntu 15.10, AMD, compiled with GCC 5.2.1.

--- a/test/public_server.c
+++ b/test/public_server.c
@@ -1079,14 +1079,14 @@ START_TEST(test_request_handlers)
 
 	ck_assert_str_eq(ri->uri, "200");
 	i = mg_read(client_conn, buf, sizeof(buf));
-	ck_assert_int_ge(i, strlen(expected_cgi_result));
+	ck_assert_int_ge(i, (int)strlen(expected_cgi_result));
 	if ((i >= 0) && (i < (int)sizeof(buf))) {
 		while ((i > 0) && ((buf[i - 1] == '\r') || (buf[i - 1] == '\n'))) {
 			i--;
 		}
 		buf[i] = 0;
 	}
-	ck_assert_int_eq(i, strlen(expected_cgi_result));
+	ck_assert_int_eq(i, (int)strlen(expected_cgi_result));
 	ck_assert_str_eq(buf, expected_cgi_result);
 	mg_close_connection(client_conn);
 #else

